addons/web_editor/static/src/xml/editor.xml

```diff
@@ -81,7 +69,7 @@
 
     <!-- Image choosing part of the Media Dialog -->
     <t t-name="web_editor.dialog.image">
-        <form class="form-inline"
+        <form class="form-inline card"
```

Comment:
Pretty sure a form-inline card with a container inside and no card-body is not valid bootstrap :wink: https://getbootstrap.com/docs/4.1/getting-started/introduction/ :)